src/libraries/Microsoft.PowerFx.Core/Texl/Builtins/Index.cs

```diff
@@ -0,0 +1,65 @@
+﻿// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT license.
+
+using System.Collections.Generic;
+using Microsoft.PowerFx.Core.App.ErrorContainers;
+using Microsoft.PowerFx.Core.Binding;
+using Microsoft.PowerFx.Core.Localization;
+using Microsoft.PowerFx.Core.Syntax.Nodes;
+using Microsoft.PowerFx.Core.Types;
+using Microsoft.PowerFx.Core.Utils;
+
+namespace Microsoft.PowerFx.Core.Texl.Builtins
+{
+    // IndexT(source:*, [n])
+    internal sealed class IndexTFunction : FunctionWithTableInput
+    {
+        public override bool IsSelfContained => true;
+
+        public override bool SupportsParamCoercion => false;
+
+        public IndexTFunction()
+            : base(
+                  "IndexT",
```

Comment:
The T suffix is used when we have functions that have both a scalar and table overload, like Log. 
In those cases, the C# class name, and the JS runtime function get the _T suffix. The string name of the function does not, and we do overload resolution based on the argument types to pick which function is actually called. 

In this case, it's a function that applies to tables, and only tables, so there's no need for a T suffix anywhere.

Reply:
removed T suffix